lib/misc/cells.js

```diff
@@ -0,0 +1,64 @@
+(function() {
+  var scopes, values;
+
+  scopes = require('./scopes');
+
+  values = require('underscore-plus').values;
+
+  module.exports = {
```

Comment:
I think I must've been using `use 'babel'` instead of `'use babel'`. Updated to the `export function` style now. `getRange` is not exported in the end.